cmd/docker/builder.go

```diff
@@ -116,10 +116,13 @@ func processBuilder(dockerCli command.Cli, cmd *cobra.Command, args, osargs []st
 		envs = append([]string{"BUILDX_BUILDER=" + dockerCli.CurrentContext()}, envs...)
 	}
 
+	// overwrite the command path for this plugin using the alias name.
+	cmd.Annotations[pluginmanager.CommandAnnotationPluginCommandPath] = fmt.Sprintf("%s %s", cmd.CommandPath(), alias)
```

Comment:
> According to the otel docs, the value of the OTEL_RESOURCE_ATTRIBUTES env var must follow the W3C Baggage format.
> 
> From what i see i think we shouldn't really include whitespace (or am I misunderstanding something?).
> 
> Does this DefaultEncoder encode in a way that is also compatible with sending the values via env var?

To clarify this doubt, i noticed the space in the value set here and in the default case of the `getPluginResourceAttributes` func above